readme.md

```diff
@@ -57,10 +56,11 @@ Recursively sort keys, including keys of objects inside arrays.
 ##### compare
 
 Type: `Function`
+Default: [Compare function.](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort)
 
-[Compare function.](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort)
-
+If given, the function signature `(a, b, object)` gets a third argument which is the current object, so that the values can be used for comparison as well, as `object[a]` and `object[b]`.
```

Comment:
```suggestion
If given, the function signature `(a, b, object)` receives a third argument which is the current object, so that the values can be used for the comparison as well, as `object[a]` and `object[b]`.
```